provisioner/clusterpy.go

```diff
@@ -496,6 +496,8 @@ func createOrUpdateEtcdStack(
 	logger.Debugf("Uploaded generated files to %s", s3Path)
 	values[s3GeneratedFilesPathValuesKey] = s3Path
 
+	values["instance_type_info"] = instanceTypes
```

Comment:
Why not keep the key name `etcd_instance_type_info` for clarity? There is already a key for the `values` map: `instanceInfoKey` (== `"InstanceInfo"`), but this one does not refer to etcd instances.

Reply:
Thanks for clarifying. On the naming, now that I understand the purpose better, I don't really have a much better suggestion :/